components/section/Project.jsx

```diff
@@ -0,0 +1,8 @@
+export default function Project(){
+    return(
+        <>
+        
+<div class="">Dacade</div>
```

Comment:
I think you can use className instead of class. also, remember to do not left empty className.